icu4c/source/test/fuzzer/collator_compare_fuzzer.cpp

```diff
@@ -10,23 +11,36 @@
 
 IcuEnvironment* env = new IcuEnvironment();
 
+static const std::array<icu_74::Collator::ECollationStrength, 5> kStrength = {
+    icu::Collator::PRIMARY,
+    icu::Collator::SECONDARY,
+    icu::Collator::TERTIARY,
+    icu::Collator::QUATERNARY,
+    icu::Collator::IDENTICAL
+};
+
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   UErrorCode status = U_ZERO_ERROR;
 
   if (size < 2)
     return 0;
 
+  uint16_t rnd = *(reinterpret_cast<const uint16_t *>(data));
```

Comment:
This is likely an unaligned read. Please use memcpy to copy the first two bytes into rnd.

Reply:
I follow the example from 
https://github.com/unicode-org/icu/blob/main/icu4c/source/test/fuzzer/break_iterator_fuzzer.cpp#L27
https://github.com/unicode-org/icu/blob/main/icu4c/source/test/fuzzer/number_format_fuzzer.cpp#L24

Reply:
Well, for a few years it seemed like people didn't care as much because Intel CPUs were lenient. More recently I have seen people chasing alignment issues again with sanitizers and claiming undefined behavior.